ADO.NET/GettingStarted/README.md

```diff
@@ -0,0 +1,11 @@
+## Overview
```

Comment:
This README feels a bit empty; I'm not sure what else to add to it that wouldn't be redundant with the other 2 READMEs in the ADO.NET directory.